frontend/nuxt.config.ts

```diff
@@ -48,49 +28,14 @@ const favicons = [
   },
 ]
 
-// Default html head
-const head = {
-  title: "Openly Licensed Images, Audio and More | Openverse",
-  meta,
-  link: [
-    ...favicons,
-    {
-      rel: "preconnect",
-      href: env.apiUrl,
-      crossorigin: "",
-    },
-    {
-      rel: "dns-prefetch",
-      href: env.apiUrl,
-    },
-    {
-      rel: "search",
-      type: "application/opensearchdescription+xml",
-      title: "Openverse",
-      href: "/opensearch.xml",
-    },
-  ],
-}
+const isProd = import.meta.env.NODE_ENV === "production"
```

Comment:
Can we change the name of this variable? `isProd` makes it sound like the deployment environment, but it's actually the type of runtime. Maybe `isDistributableBuild`, or something? This confusion is evident in this file, even, where sentry's `environment` is set to `environment: isProd ? "production" : "local",`, which is incorrect. It should match `deploymentEnv: process.env.DEPLOYMENT_ENV ?? "local",`.